pages/lists/[listId]/activity.tsx

```diff
@@ -94,6 +137,13 @@ const ListActivityPage = ({ list, numberOfContributors, isError, activityData }:
               isLoading={isLoading}
             />
           </ClientOnly>
+          <ContributionsTreemap
+            setLevel={setLevel}
+            level={level}
+            onClick={onHandleClick}
+            data={treemapData}
+            color="hsla(21, 90%, 48%, 1)"
```

Comment:
Can we generate some random colors per project instead of a single one?

Reply:
Yeah, for sure. We grab max 20 for now I think so I can get ChatGPT to give me 20.

Reply:
Done. See [`aa2d58b` (#1853)](https://github.com/open-sauced/app/pull/1853/commits/aa2d58bcd2bbad681e7a9a23cb1c04f078edfde0)